internal/services/containerapps/container_app_environment_resource.go

```diff
@@ -27,6 +27,7 @@ type ContainerAppEnvironmentModel struct {
 	LogAnalyticsWorkspaceId     string                 `tfschema:"log_analytics_workspace_id"`
 	InfrastructureSubnetId      string                 `tfschema:"infrastructure_subnet_id"`
 	InternalLoadBalancerEnabled bool                   `tfschema:"internal_load_balancer_enabled"`
+	DaprAIConnectionString      string                 `tfschema:"dapr_ai_connection_string"`
```

Comment:
Can we make this long hand for clarity?
```suggestion
	DaprAIConnectionString      string                 `tfschema:"dapr_application_insights_connection_string"`
```

Reply:
Sure. Will do.